backend/app/services/knowledge/summary_service.py

```
@@ -0,0 +1,786 @@
+
+        # Lookup model based on type
+        if model_type == "public":
+            # Query public model from Kind table (user_id=0)
+            public_model = (
+                self.db.query(Kind)
+                .filter(
+                    Kind.user_id == 0,
+                    Kind.kind == "Model",
+                    Kind.name == model_name,
+                    Kind.namespace == "default",
+                    Kind.is_active == True,
```

Ruff (0.14.11)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)